### ISSUE TITLE
False positive for placeholder selector in selector list in selector-type-case

### ISSUE THREAD

**Issue by xi** [stylelint/stylelint#2359]
I found a bug with `selector-type-case` and SCSS placeholder selectors that only seems to occur when they are used in a selector group.

> Which rule, if any, is this issue related to?

`selector-type-case`

> What CSS is needed to reproduce this issue?

`.foo, %foo {}`

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "selector-type-case": "upper"
  }
}
```

> Which version of stylelint are you using?

7.8.0

> How are you running stylelint: CLI, PostCSS plugin, Node API?

`stylelint --syntax scss ...`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Yes, it is related to [SCSS placeholder selectors](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#placeholder_selectors_)

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

```
Expected "%foo" to be "%FOO"   selector-type-case
```

**Comment by evilebottnawi** [comment]
@xi Thanks for feedback and PR

**Comment by jeddy3** [comment]
Done in https://github.com/stylelint/stylelint/issues/2360

@xi Thank you!
